features/app/components/AbstractApp.js

```diff
@@ -22,6 +31,30 @@ export class AbstractApp extends Component {
     }
 
     /**
+     * Init lib-jitsi-meet and create local participant when component is going
+     * to be mounted.
+     *
+     * @inheritdoc
+     */
+    componentWillMount() {
```

Comment:
Well, ```componentWillMount``` is earlier than ```componentDidMount``` (which happens after initial render) and also it better aligns with ```componentWiilUnmount``` method )).
But we already have ```roomSet``` dispatch in constructor, so we can actually move other dispatches there as well.